src/EnergyPlus/SetPointManager.cc

```diff
@@ -3973,7 +4271,7 @@ void InitSetPointManagers(EnergyPlusData &state)
         if (state.dataSetPointManager->InitSetPointManagersOneTimeFlag) {
 
             // "SetpointManager:SingleZone:Heating"
-            cSetPointManagerType = managerTypeName(SetPointManagerType::SZHeating);
+            cSetPointManagerType = "SetpointManager:SingleZone:Heating";
```

Comment:
Since we deleted the managerTypeName function in the header file as requested, we replaced it with the name here.

Reply:
Thanks for doing a lot of extra cleanup work beyond the scope of the new SPM, but the general intent with the enums is to define literal stings only once. I see these names are already in an array. This would be better as  `cSetPointManagerType = managerTypeName[static_cast<int>(SetPointManagerType::SZHeating]`